tests/data/channel/normalize.py

```diff
@@ -4,6 +4,8 @@
 # --- import --------------------------------------------------------------------------------------
 
 
+import random
```

Comment:
Not a huge fan of random in tests... Can cause stochastic, hard to track down problems (see clip)